crates/bevy_pbr/src/meshlet/cull_meshlets.wgsl

```diff
@@ -86,6 +105,16 @@ fn cull_meshlets(@builtin(global_invocation_id) cluster_id: vec3<u32>) {
     atomicOr(&meshlet_occlusion[cluster_id.x / 32u], occlusion_bit);
 }
 
+// https://stackoverflow.com/questions/21648630/radius-of-projected-sphere-in-screen-space/21649403#21649403
+fn lod_error_is_imperceptible(cp: vec3<f32>, r: f32) -> bool {
+    let d2 = dot(cp, cp);
+    let r2 = r * r;
+    let sphere_diameter_uv = view.projection[0][0] * r / sqrt(d2 - r2);
```

Comment:
```suggestion
    let sphere_diameter_uv = view.projection[0][0] * r * inverseSqrt(d2 - r2);
```

Reply:
The spec says inverseSqrt() does not work if e <= 0. Need to think if this is an issue.

Reply:
i dont think sqrt is guaranteed to work for e <= 0 either. I'd just slab an abs or max 0 on it if its a worry, being explicit about edge case behavior intent is valuable and saturating/clamping is often free in gpus